### PR TITLE
Add e-mail support button to docs footer

### PR DESCRIPTION
**Pull request by neynah**
"E-mail support" button added to the footer. Fixes: #2517 

image

### CODE REVIEW

**Comment by kentonv** [comment]
Hmm. Given that we're not full-time anymore, I think we should direct people towards sandstorm-dev / IRC / other community options and deprecating support@ (which is pretty much now an alias for me personally). Maybe we could link to `sandstorm.io/community` here or maybe we need a better landing page specifically for "where do I ask questions"?

**Comment by neynah** [comment]
I think sandstorm-dev & IRC make the most sense for the time being until we have a better "where do I ask questions?" landing page as you mention. Perhaps this can live somewhere like sandstorm.io/help.

**Comment by neynah** [comment]
Updated the commit to display "Sandstorm-dev Group" and "IRC"

image

File: docs/sandstorm_readthedocs_modified_theme/base.html [hunk 1]
```diff
@@ -104,6 +104,8 @@
           </div>
 	  {%- block footer %}
           {% include "footer.html" %}
+          <div id="get-help">Need help?<a href="https://groups.google.com/forum/#!forum/sandstorm-dev" class="btn btn-neutral">Sandstorm-dev Group</a> <a href="https://kiwiirc.com/client/irc.freenode.net/?channel=#sandstorm" class="btn btn-neutral">IRC</a>
++         </div>
```
Comment (kentonv) [inline docs/sandstorm_readthedocs_modified_theme/base.html]: Oops, there's still a stray `+` here which shows up on the final site as a "+" to the right of the buttons.

Comment (kentonv) [inline docs/sandstorm_readthedocs_modified_theme/base.html]: I'll fix it.

Comment (neynah) [inline docs/sandstorm_readthedocs_modified_theme/base.html]: Oops! Thanks.

**Pull request merged**